internal/app/machined/internal/phase/config/extra_files.go

```diff
@@ -82,3 +103,22 @@ func (task *ExtraFiles) runtime(r runtime.Runtime) (err error) {
 
 	return result.ErrorOrNil()
 }
+
+func existsAndIsFile(p string) (err error) {
+	var info os.FileInfo
+
+	info, err = os.Stat(p)
```

Comment:
this logic is a bit off to satisfy both create and overwrite requests

Reply:
fixed